optaplanner-examples/src/test/java/org/optaplanner/examples/common/app/SolverPerformanceTest.java

```diff
@@ -43,13 +51,40 @@
  * Always use a {@link Test#timeout()} on {@link Test}, preferably 10 minutes because some of the Jenkins machines are old.
  * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
  */
+@RunWith(Parameterized.class)
 public abstract class SolverPerformanceTest<Solution_> extends LoggingTest {
 
-    private static final String MOVE_THREAD_COUNT_OVERRIDE = System.getProperty(TestSystemProperties.MOVE_THREAD_COUNT);
+    private static final String MOVE_THREAD_COUNTS_VALUE = System.getProperty(TestSystemProperties.MOVE_THREAD_COUNTS);
```

Comment:
variable naming: I wouldn't end a variable name with "VALUE" because they are always value. I'd probably make it MOVE_THREAD_COUNTS_STRING (to signal it's not yet parsed) or just MOVE_THREAD_COUNTS.